docs/user-guide/configuring.md

```diff
@@ -712,6 +712,61 @@ module.exports = {
 }
 ```
 
+## Configuration Based on Glob Patterns
+
+Sometimes a more fine-controlled configuration is necessary, for example if the configuration for files within the same directory has to be different. Therefore you can provide configurations under the `overrides` key that will only apply to files that match specific glob patterns, using the same format you would pass on the command line (e.g., `app/**/*.test.js`).
```

Comment:
I think you'll need to update the [config schema](https://github.com/eslint/eslint/blob/c70b0eda8794e552bcbfccbc0d5f46dfb4ea7783/conf/config-schema.json) because ESLint now throws f it encounters an unknown top-level property.